modules/boulder/src/test/ts/atomic/api/FieldSchemaTest.ts

```diff
@@ -15,6 +15,7 @@ UnitTest.test('Atomic Test: api.FieldSchemaTest', () => {
 
     StructureSchema.asRaw('spec', schema, input).fold(
       (err) => {
+        // eslint-disable-next-line @typescript-eslint/only-throw-error
```

Comment:
Is there similar rule to not reject a promise with something that is not an instance of error that would be useful as well. Also I think we should try to fix this one if we can, maybe not in this PR.

Reply:
I have created https://ephocks.atlassian.net/browse/TINY-11766 as a follow up